corva/models/task.py

```diff
@@ -44,3 +46,9 @@ class TaskEventData(BaseEventData):
     id: Optional[str] = None
     task_id: str
     version: conint(ge=2, le=2)  # only utils API v2 supported
+
+
+class TaskEvent(BaseEvent, TaskEventData):
```

Comment:
Tasks can have arbitrary payloads that would be fitted into TaskEventData. Is that accounted for here?

Reply:
yes, `TaskEventData` inherits from `BaseEventData` which allows arbitrary fields [link](https://github.com/corva-ai/python-sdk/blob/a715bf4d4cffe06f1794c20fc6e6ddc83c96a8ae/corva/models/base.py#L37)